### PR TITLE
Add composer version to issue template

### PR DESCRIPTION
**Pull request by dakinggg**
# What does this PR do?
Adds composer version to the requested environment information in a bug report.

### CODE REVIEW

**Review by mvpatel2000 — APPROVED** [review]
Nice addition :)

File: .github/ISSUE_TEMPLATE/---bug-report.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ assignees: ''
 ** Environment **
 <!-- Provide the below as necessary -->
 
+- composer version: [dev]
```
Comment (bandish-shah) [inline .github/ISSUE_TEMPLATE/---bug-report.md:13]: To collect environment information can we just standardize on asking the user to run `composer_collect_env`?

Comment (dakinggg) [reply .github/ISSUE_TEMPLATE/---bug-report.md:13]: oh yes, i did not know that existed.

Comment (bandish-shah) [reply .github/ISSUE_TEMPLATE/---bug-report.md:13]: FYI here's an example of a report:
```
root@interactive-a100-40gb-1-d7cy-w4thr:/# composer_collect_env
Collecting system information...
---------------------------------
System Environment Report
Created: 2022-10-20 20:30:26 UTC
---------------------------------

PyTorch information
-------------------
PyTorch version: 1.12.1+cu116
Is debug build: False
CUDA used to build PyTorch: 11.6
ROCM used to build PyTorch: N/A

OS: Ubuntu 20.04.4 LTS (x86_64)
GCC version: (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
Clang version: Could not collect
CMake version: Could not collect
Libc version: glibc-2.31

Python version: 3.9.14 (main, Sep  7 2022, 23:43:29)  [GCC 9.4.0] (64-bit runtime)
Python platform: Linux-5.4.0-126-generic-x86_64-with-glibc2.31
Is CUDA available: True
CUDA runtime version: 11.6.124
GPU models and configuration: GPU 0: NVIDIA A100-SXM4-40GB
Nvidia driver version: 510.47.03
cuDNN version: Probably one of the following:
/usr/lib/x86_64-linux-gnu/libcudnn.so.8.4.0
/usr/lib/x86_64-linux-gnu/libcudnn_adv_infer.so.8.4.0
/usr/lib/x86_64-linux-gnu/libcudnn_adv_train.so.8.4.0
/usr/lib/x86_64-linux-gnu/libcudnn_cnn_infer.so.8.4.0
/usr/lib/x86_64-linux-gnu/libcudnn_cnn_train.so.8.4.0
/usr/lib/x86_64-linux-gnu/libcudnn_ops_infer.so.8.4.0
/usr/lib/x86_64-linux-gnu/libcudnn_ops_train.so.8.4.0
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

Versions of relevant libraries:
[pip3] numpy==1.22.4
[pip3] pytorch-ranger==0.1.1
[pip3] torch==1.12.1+cu116
[pip3] torch-optimizer==0.3.0
[pip3] torchmetrics==0.7.3
[pip3] torchtext==0.13.1
[pip3] torchvision==0.13.1+cu116
[pip3] vit-pytorch==0.35.8
[conda] Could not collect


Composer information
--------------------
Composer version: 0.10.1
Composer commit hash: None
Host processor model name: AMD EPYC 7513 32-Core Processor
Host processor core count: 64
Number of nodes: 1
Accelerator model name: NVIDIA A100-SXM4-40GB
Accelerators per node: 1
CUDA Device Count: 1
```

**Review by bandish-shah — APPROVED** [review]
LGTM

**Pull request merged**